contracts/core/tokens/rebalancing-v2/RebalancingSetState.sol

```diff
@@ -187,6 +193,20 @@ contract RebalancingSetState {
         manager = _newManager;
     }
 
+    function setEntryFee(
```

Comment:
Can you add this to IRebalancingSetTokenV2 as well

Reply:
sure